src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/CountFunctions.txt

```diff
@@ -23,4 +23,7 @@ Blank()
 #Error
 
 >> CountIf([1, 2, 3, 4], 123)
-Errors: Error 22-25: Invalid argument type (Number). Expecting a Boolean value instead.
+Errors: Error 22-25: Expected boolean. We expect a boolean (true/false) at this point in the formula.|Error 0-26: The function 'CountIf' has some invalid arguments.
```

Comment:
consider setting haspreciseerrors on the function, the 2nd error message isn't super useful

Reply:
Added

Reply:
what about lookup?